doc/InfiniTimeVision.md

```diff
@@ -0,0 +1,32 @@
+# InfiniTime Vision
+
+The purpose of this document is to steer efforts towards a common goal, and as such should be taken into consideration with all developments.
+
+## InfiniTime
+
+InfiniTime is a community-built smartwatch firmware, providing freedom and privacy that's usually insufficient in wearable technology.
```

Comment:
@Danimations has suggested to rephrase this as:
```suggestion
InfiniTime is a community-built firmware for the PineTime smartwatch. It offers freedom and privacy advantages unavailable to users of other proprietary wearable technology.
```

Perhaps saying it's only for the PineTime is too specific however, as there will be ports in the future.

Reply:
I personally think that what is already there is mostly good, but that specifying that the other wearable technology it mentions is the proprietary kind would improve it.

Reply:
Given your feedback, perhaps this is appropriate:

"InfiniTime is a community-built smartwatch firmware. It offers freedom and privacy advantages unavailable to users of proprietary wearable technology."